.circleci/config.yml

```diff
@@ -59,6 +63,22 @@ jobs:
                   echo "about to publish to tag ${CIRCLE_TAG}"
                   ls -l *
                   ghr -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} ${CIRCLE_TAG} ./artifacts/honeycomb-opentracing-proxy
+  publish-docker-tag:
+    executor: docker-publisher
+    steps:
```

Comment:
please use a buildevents `with_job_span` so that this step gets included in the trace.